fe/fe-core/src/main/java/com/starrocks/catalog/OlapTable.java

```diff
@@ -1844,6 +1845,21 @@ public TStorageFormat getStorageFormat() {
         return tableProperty.getStorageFormat();
     }
 
+    public void setCompressionType(TCompressionType compressionType) {
```

Comment:
Does the compressType can be modified using alter table?

Reply:
alter table is currently not supported.